ansible/deploy_proxy.yml

```diff
@@ -96,7 +128,7 @@
         tasks_from: custom_site
```

Comment:
Seems like we could get rid of this task file since it's just duplicating work done elsewhere. I know I created that file (https://github.com/dimagi/commcare-cloud/commit/b2c3b5a458c4e0aa3492ad93e0779afbeb356336) but I don't see why we can't treat them the same as the other `special_sites`.

I think it was like this because they use a separate inventory group but I don't see the necessity for that.

Reply:
Okay, it's good to hear that because I was scratching my head trying to understand why that needed a different task file. I think I'll address this in a followup PR, since each iteration on this stuff takes a bit of manual testing.